media/Config/vscode.css

```diff
@@ -0,0 +1,301 @@
+:root {
+  --container-margin: 10px;
```

Comment:
When I run css lint (from draft #113)
I got these errors
```
vscode.css
2: error at line 2, col 3
Expected RBRACE at line 2, col 3.
  --container-margin: 10px;

vscode.css
3: error at line 3, col 3
Expected RBRACE at line 3, col 3.
  --container-padding: 20px;
```
Is it OK?

Reply:
CSSlint doesn't support CSS custom properties. but it works fine in browser.

Reply:
> CSSlint doesn't support CSS custom properties. but it works fine in browser.

This means, I cannot enable csslint? hum...

Reply:
@mhs4670go , any thoughts on this?

Reply:
oh... I didn't take that into account. I'll fix it...